packages/office-addin-dev-settings/src/commands.ts

```diff
@@ -61,7 +65,9 @@ export async function clear(manifestPath: string) {
     await devSettings.clearDevSettings(manifest.id!);
 
     console.log("Developer settings have been cleared.");
+    usageDataObject.reportSuccess("clear");
```

Comment:
can we make this "clearDevSettings" so it's more obvious what method it's related to in our usage data?

Reply:
It is the "clear" command for office-addin-dev-settings package. Which is provided in the data. Is this different than "start" for debugging or "webview" command?